src/collection.js

```diff
@@ -169,14 +169,14 @@ export default class Collection {
    * @param  {Object} options
    * @return {Promise}
    */
-  create(record, options={synced: false}) {
+  create(record, options={forceUUID: false, synced: false}) {
```

Comment:
I debated this, but I couldn't see any use case for end users to use this option, which I've added for test context creation only. Your call.